app/Filament/Resources/GameResource/RelationManagers/AchievementSetsRelationManager.php

```diff
@@ -197,38 +199,26 @@ public function table(Table $table): Table
                             ]),
                     ])
                     ->action(function (array $data) use ($game) {
-                        // Add a new game_achievement_set row. The existing row stays
-                        // New row is basically the same as the old row, but the game_id and type are different values.
-                        // It'll use the same achievement_set_id.
+                        try {
```

Comment:
There's some protection built into this search to prevent users from attaching the wrong things as subsets.

First and foremost, only subsets with the same system ID as the current game you're viewing are shown:
```php
Game::where('Title', 'like', "%[Subset -%")
    ->where('ConsoleID', $game->ConsoleID)
    ->where('achievements_published', '>', 0)
```

For game 28275, the list is only populated with a single item because that's the only standalone subset that has any published achievements.

For game 764, searching for "final" only shows two results because those are the only GBA subsets with the word "final" in their name.

For the case where you typed "happ" and got mixed results, I assume that's because Filament's `->searchable()` attribute is actually doing a client-side fuzzy search from the server-side fetched results. Pokemon Sapphire Version ranks 1st because it scores highest in the fuzzy search criteria.